From 355e8b6bd704cb57de4f72feebb75d3d977b075d Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Fri, 27 Jan 2023 19:44:15 -0800
Subject: [PATCH 04/18] shm: Avoid unused variable warning !HAVE_MEMFD_CREATE
 case

../src/wayland-shm.c:307:14: warning: unused variable 'statbuf' [-Wunused-variable]
        struct stat statbuf;
                    ^
../src/wayland-shm.c:308:6: warning: unused variable 'seals' [-Wunused-variable]
        int seals;
            ^
2 warnings generated.

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
---
 src/wayland-shm.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/wayland-shm.c b/src/wayland-shm.c
index 3ac4add..8d27df5 100644
--- a/src/wayland-shm.c
+++ b/src/wayland-shm.c
@@ -357,8 +357,10 @@ shm_create_pool(struct wl_client *client, struct wl_resource *resource,
 		uint32_t id, int fd, int32_t size)
 {
 	struct wl_shm_pool *pool;
+#ifdef HAVE_MEMFD_CREATE
 	struct stat statbuf;
 	int seals;
+#endif
 	int prot;
 	int flags;
 	uint32_t version;
-- 
2.24.3 (Apple Git-128)

